app/views/migration/_form.html.erb

```diff
@@ -42,6 +42,18 @@
           <% end %>
         <% end %>
       <% end %>
+
+      <%= agg.nested_fields_for(:edm_hasViews, legend: i18n.t('legends.each_object', number: '')) do |wr| %>
+        <%= wr.remove_nested_fields_link(i18n.t('buttons.remove_object')) %>
+        <%= wr.input(:media, label: false) %>
```

Comment:
Could we add a hint to all web resource media fields indicating permitted content types?

Reply:
done, although the nested form hint had to be added inline because the .yml file doesn't work with the array of ids